Jenkinsfile

```diff
@@ -1,14 +1,13 @@
 // Tell Jenkins how to build projects from this repository
 pipeline {
-	agent {
-		label 'java7'
-	} 
+	agent any 
 	parameters {
-		booleanParam(defaultValue: true, description: '''This parameter is used to allow not to execute Sonar analysis. It is safe to always make this true, as the Sonar-trigger job will trigger this job without the SKIP_SONAR parameter set daily.''', name: 'SKIP_SONAR')
-        choice(choices: 'ci\nintegration\nrelease', description: '', name: 'BUILD_TYPE')
+		string(name: 'VIATRA_REPOSITORY', defaultValue: 'https://hudson.eclipse.org/viatra/job/viatra-master/lastSuccessfulBuild/artifact/releng/org.eclipse.viatra.update/target/repository')
```

Comment:
Is this parameter required for master builds? I'd hazard no - we should rely on the latest VIATRA version instead...

Reply:
I guess this means the latest VIATRA version (last successful build). Did you mean latest released version of VIATRA?

Reply:
This parameter was introduced to allow testing specific VIATRA builds (even those created Gerrit changes) without modifying Massif code (specifically cyclic linking problems at some point).